_posts/2024-02-26-cache-refresh.md

```diff
@@ -0,0 +1,166 @@
+any open source project without active maintenance becomes stale. And like a
+stale cache, a stale open source project can cause issues, or at a minimum
+becomes outdated and unsuitable for modern use. Staleness and other problems
+meant that RubiX had to go.
+
+## Making progress
+
+Time moved on and Trino now includes powerful connectors for the modern
+lakehouse formats Delta Lake, Hudi, and Iceberg:
+
+<div class="container">
+  <div class="row">
+    <div class="col-sm">
+      <a href="{{site.url}}/docs/current/connector/delta-lake.html" target="_blank">
+        <img src="{{site.url}}/assets/images/logos/delta-lake.png" title="Delta Lake connector">
+      </a>
+    </div>
+    <div class="col-sm">
+      <a href="{{site.url}}/docs/current/connector/hudi.html" target="_blank">
+        <img src="{{site.url}}/assets/images/logos/apache-hudi.png" title="Hudi connector">
+      </a>
+    </div>
+    <div class="col-sm">
+      <a href="{{site.url}}/docs/current/connector/iceberg.html" target="_blank">
+        <img src="{{site.url}}/assets/images/logos/apache-iceberg.png" title="Iceberg connector">
+      </a>
+    </div>
+  </div>
+</div>
+
+Hive is still around, just like HDFS, but we consider them both close to legacy
+status. Yet all four connectors could benefit from caching. Good news came at
+Trino Summit 2022 when Hope Wang and Beinan Wang from
+[Alluxio]({{site.url}}/ecosystem/add-on.html#alluxio) presented about their
+integration with Trino and the Hive connector - [Trino optimization with
+distributed caching on data lake]({% post_url
+2023-07-21-trino-fest-2023-alluxio-recap %}). They mentioned plans to open
+source their implementation and an initial PR was created.
+
+<div class="container">
+  <div class="row">
+    <div class="col-sm"></div>
+    <div class="col-sm">
+      <img src="{{site.url}}/assets/images/logos/alluxio.png" title="Alluxio">
+    </div>
+    <div class="col-sm"></div>
+  </div>
+</div>
+
+## Collaboration
+
+The initial presentation and pull request (PR) planted a seed in the community.
+The Trino project had been moving fast in terms of deprecating the old
+dependencies from the Hadoop and Hive ecosystem so the Alluxio PR was not ready
+for a merge. Discussions with [David Phillips](https://github.com/electrum) laid
+out the path to adjust to the new file system support.
+
+In the end it was [Florent Delannoy](https://github.com/pluies) who started
+another [PR for file system caching support, specifically for the Delta Lake
+connector](https://github.com/trinodb/trino/pull/18719). His team mate [Jonas
+Irgens Kylling](https://github.com/jkylling), also a [presenter from Trino Fest
+  2023]({% post_url 2023-07-14-trino-fest-2023-dune %}), took over the work on
+the PR. The collaboration on it was an **epic effort**. Over 300
+comments directly on GitHub and numerous hours of coding, reviewing, testing,
+and discussion on slack and elsewhere finally ended with a merge after many
```

Comment:
```suggestion
and discussion on Slack and elsewhere finally ended with a merge after many
```
"ended with a merge" feels anticlimactic, maybe specify that this means the launch of the feature and it shipping live with Trino

Reply:
Reworded